### PR TITLE
Add core files.

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "jsonld-document-loader",
+  "version": "0.0.1-0",
+  "description": "A Decentralized Identifier utility library for Veres One",
```
Comment (davidlehn) [inline package.json]: Fix

Comment (mattcollier) [reply package.json]: Done, ty.

File: lib/JsonLdDocumentLoader.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ */
+'use strict';
+
+class JsonLdDocumentLoader {
+  constructor() {
+    this.documents = new Map();
+  }
+
+  addStatic(url, document) {
+    this.documents.set(url, document);
```
Comment (dlongley) [inline lib/JsonLdDocumentLoader.js:18]: Could add a check that `document` is not `null` and is an object, else throw a `TypeError`.

**Review by dlongley — APPROVED** [review]

File: lib/JsonLdDocumentLoader.js (hunk 2 above)
Comment (mattcollier) [reply lib/JsonLdDocumentLoader.js:18]: Added validation and tests.

**Pull request merged**